app/controllers/application_pet_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationPetController < ApplicationController
+  def create
```

Comment:
Great work pulling this action into its own controller for joining Applications with Pets!